lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -385,13 +384,8 @@ def linodeServers(module, api, state, name, alert_bwin_enabled, alert_bwin_thres
                 # Update linode Label to match name
                 api.linode_update(LinodeId=linode_id, Label='%s_%s' % (linode_id, name))
                 # Update Linode with Ansible configuration options
-                api.linode_update(LinodeId=linode_id, ALERT_BWIN_ENABLED=alert_bwin_enabled,
-                        ALERT_BWIN_THRESHOLD=alert_bwin_threshold, ALERT_BWOUT_ENABLED=alert_bwout_enabled,
-                        ALERT_BWOUT_THRESHOLD=alert_bwout_threshold, ALERT_BWQUOTA_ENABLED=alert_bwquota_enabled,
-                        ALERT_BWQUOTA_THRESHOLD=alert_bwquota_threshold, ALERT_CPU_ENABLED=alert_cpu_enabled,
-                        ALERT_CPU_THRESHOLD=alert_cpu_threshold, ALERT_DISKIO_ENABLED=alert_diskio_enabled,
-                        ALERT_DISKIO_THRESHOLD=alert_diskio_threshold, BACKUPWEEKLYDAY=backupweeklyday,
-                        BACKUPWINDOW=backupwindow, LPM_DISPLAYGROUP=displaygroup, WATCHDOG=watchdog)
+                api.linode_update(LinodeId=linode_id,
+                        LPM_DISPLAYGROUP=displaygroup, WATCHDOG=watchdog, **kwargs)
```

Comment:
I can move them to kwargs at this point. I'm open to either way. Originally my goal was as few changes as possible to just fix bugs, then saw the behavior was completely different and had to rewrite to the splatting to get desired results.